### PR TITLE
Adding net8 support

### PR DESCRIPTION
**Pull request by MartyMcFlaye**
Quick pull request, in order to support net8 framework

### CODE REVIEW

File: tests/Younited.HealthCheck.EntityFrameworkMigrations.Tests/Younited.HealthCheck.EntityFrameworkMigrations.Tests.csproj [hunk 1]
```diff
@@ -1,19 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net7.0</TargetFrameworks>
+    <TargetFrameworks>net8.0</TargetFrameworks>
```
Comment (Greybird) [inline tests/Younited.HealthCheck.EntityFrameworkMigrations.Tests/Younited.HealthCheck.EntityFrameworkMigrations.Tests.csproj]: Shouldn't we run tests against net6.0, net7.0, and net8.0 to ensure the frameworks for which we publish the library are actually tested ?
To be fair, the situation is not new, but maybe it is the right time to ask ourselves what to do with this regards ?

Comment (MartyMcFlaye) [reply tests/Younited.HealthCheck.EntityFrameworkMigrations.Tests/Younited.HealthCheck.EntityFrameworkMigrations.Tests.csproj]: I agree.
My first try "locally" was to add net8. Then I realized net6 was not tested so I decided to fit with existing.
But i will change that.

Comment (Greybird) [reply tests/Younited.HealthCheck.EntityFrameworkMigrations.Tests/Younited.HealthCheck.EntityFrameworkMigrations.Tests.csproj]: @MartyMcFlaye, we took the opportunity with @GuillaumeGuerra to enhance this part as part of your change. Thanks!

**Review by YounitedCreditOss — APPROVED** [review]
all good

**Pull request merged**